agent/provider/src/market/negotiator/common.rs

```diff
@@ -18,12 +19,10 @@ pub enum ProposalResponse {
         offer: NewOffer,
     },
     AcceptProposal,
-    #[display(
-        fmt = "RejectProposal{}",
-        "reason.as_ref().map(|r| format!(\" (reason: {})\", r)).unwrap_or(\"\".into())"
-    )]
+    #[display(fmt = "RejectProposal (reason: {})", "reason.display()")]
```

Comment:
Some concise indication whether rejection is final would be beneficial:
`RejectProposal (F reason: None)

Reply:
I only wanted to preserve previous version. We can do this in future improvements